sdk/cosmos/azure-cosmos-kafka-connect/doc/configuration-reference.md

```diff
@@ -2,64 +2,64 @@
+| `azure.cosmos.source.database.name`               | None                     | Cosmos DB database name.                                                                                                                                                                                                                             |
+| `azure.cosmos.source.containers.includeAll`       | `false`                  | Flag to indicate whether reading from all containers.                                                                                                                                                                                                |
+| `azure.cosmos.source.containers.includedList`     | `[]`                       | Containers included. This config will be ignored if azure.cosmos.source.includeAllContainers is true.                                                                                                                                        |
+| `azure.cosmos.source.containers.topicMap`         | `[]`                       | A comma delimited list of Kafka topics mapped to Cosmos containers. For example: topic1#con1,topic2#con2. By default, container name is used as the name of the kafka topic to publish data to, can use this property to override the default config |
+| `azure.cosmos.source.changeFeed.startFrom`        | `Beginning`              | ChangeFeed Start from settings (Now, Beginning or a certain point in time (UTC) for example 2020-02-10T14:15:03) - the default value is 'Beginning'.                                                                                                 |
+| `azure.cosmos.source.changeFeed.mode`             | `LatestVersion`          | ChangeFeed mode (LatestVersion or AllVersionsAndDeletes).                                                                                                                                                                                            |
+| `azure.cosmos.source.changeFeed.maxItemCountHint` | `1000`                   | The maximum number of documents returned in a single change feed request. But the number of items received might be higher than the specified value if multiple items are changed by the same transaction.                                           |
+| `azure.cosmos.source.metadata.poll.delay.ms`      | `300000`                 | Indicates how often to check the metadata changes (including container split/merge, adding/removing/recreated containers). When changes are detected, it will reconfigure the tasks. Default is 5 minutes.                                           |
+| `azure.cosmos.source.metadata.storage.topic`      | `_cosmos.metadata.topic` | The name of the topic where the metadata are stored. The metadata topic will be created if it does not already exist, else it will use the pre-created topic.                                                                                        |
```

Comment:
should we call it `topicName` instead of just `topic`? To be consistent with `topicMap`?

Reply:
updated